lib/pure/distros.nim

```diff
@@ -18,13 +18,10 @@
 ##
 ## The above output could be the result of a code snippet like:
 ##
-## .. code-block:: nim
-##
-##   if detectOs(Ubuntu):
-##     foreignDep "lbiblas-dev"
-##     foreignDep "libvoodoo"
-##
-##
+runnableExamples:
+  if detectOs(Ubuntu):
```

Comment:
```nim
if defined(nimEnableExtraExamples) and detectOs(Ubuntu):
   ...
```
that way it won't actually run the install commands on ubuntu simply because you ran `nim doc`

and if instead of when so that `nim doc` checks that the disabled block compiles